src/components/query.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./myquery.scss";
+import { CardProduct, ICardProduct } from "./CardProduct";
```

Comment:
it's better to have your imports sorted
most common sort order
libs
local import
assets
styles